### PR TITLE
Profile Behavior: Remove bi-directional syncing - Step 1

### PR DESCRIPTION
**Pull request by DustyReagan**
Fixes https://github.com/Automattic/dotcom-forge/issues/7748

## Proposed changes:

* Revert the changes in https://github.com/Automattic/jetpack/pull/37352 and https://github.com/Automattic/jetpack/pull/37316
* We're taking a different approach: pfsHM7-1cU-p2#comment-1540

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

pfsHM7-1cU-p2#comment-1540

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Load this PR on an Atomic Classic site using Jetpack Beta Plugin instructions in the [comments below](https://github.com/Automattic/jetpack/pull/37862#issuecomment-2166596495)
* Try changing the language in /wp-admin/profile.php. It should not update Calypso /me.
* Try changing the language in Calypso /me. It should not update /wp-admin/profile.php.
* Consider if this could affect Simple sites when deployed to wpcom.
* Consider if this could create deploy time fatals when deployed to wpcom. (I _think_ this is referenced as a package on wpcom, so I don't think it is an issue.)

### CODE REVIEW

File: projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php [hunk 1]
```diff
@@ -91,8 +91,6 @@ public static function load_features() {
 		require_once __DIR__ . '/features/wpcom-block-editor/functions.editor-type.php';
 		require_once __DIR__ . '/features/wpcom-site-menu/wpcom-site-menu.php';
 		require_once __DIR__ . '/features/wpcom-themes/wpcom-themes.php';
-		require_once __DIR__ . '/features/calypso-locale-sync/sync-wp-admin-to-calypso.php';
-		require_once __DIR__ . '/features/calypso-locale-sync/sync-calypso-to-wp-admin.php';
```
Comment (DustyReagan) [inline projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php:95]: Confidence check. Removing these shouldn't cause deploy time fatals on wpcom, right? 🤔

Comment (miksansegundo) [reply projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php:95]: I got this message on other PR when removing code:

** WARNING **
It is not safe to deploy the removal of a file along with the removal of the required_once() for this file.
Please undo the file removal and deploying it separately. Alternatively, you can create a new diff to remove the required_once() function and deploy that first.

Comment (DustyReagan) [reply projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php:95]: Oh, maybe we should just remove the requires in this PR to be on the safe side.

**Review by miksansegundo — APPROVED** [review]

**Comment by miksansegundo** [comment]
The step 2 will [remove the files](https://github.com/Automattic/jetpack/pull/37862/commits/f1548e55d6ca6ea0656e60fa3157f13f55a79e1d) after this PR is deployed.

**Pull request merged**